src/Microsoft.Azure.ServiceBus/MessageReceivePump.cs

```diff
@@ -76,7 +79,7 @@ bool ShouldRenewLock()
                 catch (Exception exception)
                 {
                     MessagingEventSource.Log.MessageReceivePumpTaskException(this.messageReceiver.ClientId, string.Empty, exception);
-                    this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Receive));
+                    this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Receive, namespaceName));
```

Comment:
In that case I'll introduce some Context object rather than multiple parameters. It will also be better from a public API perspective as signature on the method won't be considered broken if additions are made.